straxen/plugins/led_cal/led_calibration.py

```diff
@@ -214,8 +222,8 @@ def get_led_windows(
     led_hit_extension. If no hit is found in the record, return the default window.
 
     :param records: Array of the records to search for LED hits.
-    :param default_window: Default window to use if no LED hit is found given as a tuple (start,
-        end)
+    :param minimum_led_position: The minimum position of the LED hits. Hits before this sample are
```

Comment:
What is the meaning of "position" here?

Reply:
The sample index of the record.

Reply:
Then shall it be "index" instead? :)

Reply:
If you think that makes it clearer, I am fine with that.

Reply:
Please change it. Thank you very much!

Reply:
Sorry for being tedious. Maybe it is better to change all "position" to "index" in `straxen/plugins/led_cal/led_calibration.py` file.

Reply:
@tflehmke do you want to make further changes? :)

Reply:
OK, since in `LEDAfterpulseProcessing` we used "position", we do not need to change this further.